unit2-java/lesson10.rst

```diff
@@ -5,7 +5,19 @@ Goals
 -----
 
 - Java development and tooling; exceptions
-   - Gradle, IDE in detail, compilation and JVM
+- Gradle, IDE in detail, compilation and JVM
 
 Objectives
 ----------
+
+- Describe what an IDE like IntelliJ offers programmers that text
+   editors do not
+- Run your program in an IDE
+- Use keyboard shortcuts in IntelliJ
+- Easily refactor and rename in IntelliJ
```

Comment:
These seem more like lesson 1 objectives rather than lesson 10.

Perhaps these should go into an appendix?  Alternatively, re-word these objectives to focus more specifically on the development of a web app.